### ISSUE TITLE
SHA256SUMS missing pivx-3.0.6.0-High-Sierra-Only.dmg

### ISSUE THREAD

**Issue by cryptoj-pivx** [PIVX-Project/PIVX#519]
### Issue
SHA256SUMS.asc does not include the hash value for pivx-3.0.6.0-High-Sierra-Only.dmg
The integrity of PIVX Core for High Sierra can only be guaranteed by building from source.

### Can you reliably reproduce the issue?
Yes

#### If so, please list the steps to reproduce below:
1. Download pivx-3.0.6.0-High-Sierra-Only.dmg and SHA256SUMS.asc
2. shasum -c SHA256SUMS.asc

### Expected behavior
Hash value should be included.

### Actual behavior
Hash value is not included.

### What version of PIVX Core are you using?
Official PIVX Core Qt version v3.0.6 (64-bit) High Sierra binary

**Comment by Fuzzbawls** [comment]
Yes, the sha256sum of the High Sierra file didn't make into the `SHA256SUMS.asc` file due to the timing of it being available.

I'll regenerate the signature file when i'm back home next week to include the HS specific file.

**Comment by danielelis** [comment]
What day next week can we expect the issue closed?
